packages/cli/commands/sandbox/create.js

```diff
@@ -79,6 +79,10 @@ exports.handler = async options => {
   }
   const sandboxType = sandboxTypeMap[type] || sandboxTypeMap[typePrompt.type];
 
+  const trackingSandboxType =
```

Comment:
I think we should try to find a pattern where we can log metadata for a command without having it look like commands are getting more usage than they actually are. Typically we try to log one user event per command and that lets us get an accurate count for total number of command runs.

Maybe we need some new [action](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/lib/usageTracking.js#L37) type other than `cli-command`. Maybe something like `cli-command-metadata`? Then we can fire that one as often as we like, and it won't interfere with any of the total counts from the `cli-command` action.

Reply:
We actually just had another meeting on this yesterday with the team! We're going to go ahead and remove all the duplicate event tracking with extra metadata and were planning on creating another `cli-command` event with a separate name to track sandbox events. 

The `cli-command-metadata` one sounds a bit cleaner though, would we have the same metadata limitations as we do for the current implementation?

Reply:
Great! That sounds like a good plan 👌 

> would we have the same metadata limitations as we do for the current implementation?

Yeah I think so. AFAIK those limitations exist everywhere in usage tracking, regardless of the action type we're using. I think we hit a company-wide limit on new field types so we have to re-use existing ones.